### DISCUSSION TITLE
Ambiguous condition - How to disable

### DISCUSSION THREAD

**Discussion by kamilblue** [Camelcade/Perl5-IDEA#2917] [Q&A]
I'm new to the PERL plugin, I started using it with pycharm community 2024.2.
An annoying warning I'm getting is "Ambiguous Condition" on a simple code, i don't see any issue with the code:
```
my $example = 0;
if ($example) {
    print "Example is TRUE\n";
} else {
    print "Example is FALSE\n";
}
```

Is there an option to disable this warning?

**Comment by hurricup** [comment]
Yes, there should be an option to disable inspection and there is an explanation on the meaning of a warning.

**Comment by kamilblue** [comment]
I found the settings to disable this checker, it took me some time :)
The setting is under: Settings -> Editor -> Inspections -> Perl5 -> Truthiness

I just unselected it and the warnings are gone.

**Reply by hurricup** [reply]
There is also a quickfix on the warning (alt-enter) with option to disable the inspection warned this particular thing.
